app/collections/lane/[slug]/page.tsx

```diff
@@ -10,8 +12,9 @@ export async function generateMetadata({
   params,
 }: LaneProps): Promise<Metadata> {
   const slug = params.slug;
+  const title = slugToString(params.slug);
   return {
-    title: `${slug} - NYPL Digital Collections`,
+    title: `${title} - NYPL Digital Collections`,
```

Comment:
As mentioned in previous PRs– we need to get the title from the endpoint, since stringifying it ourselves is not failsafe